lib/wallet/test/unit/Cardano/Wallet/Write/Tx/Balance/TokenBundleSizeSpec.hs

```diff
@@ -129,10 +153,9 @@ unit_assessTokenBundleSize_fixedSizeBundle
             , actualLengthBytes <= expectedMaxLengthBytes
             ]
   where
-    actualAssessment = assessTokenBundleSize
-        (mkTokenBundleSizeAssessor maxSize)
-        bundle
-    actualLengthBytes = computeTokenBundleSerializedLengthBytes bundle
+    actualAssessment = assessTokenBundleSize assessor bundle
+    v = eraProtVerLow @StandardBabbage -- FIXME!
```

Comment:
👀

Reply:
I think just dropping the `FIXME` is fine. Ideally we'd re-use the same version as was used by the `TokenBundleSizeAssessor` but 🤷‍♂️